TACC/Core-CMS-Custom
# fix(apcd): remove "NEW " from Getting Started

## Overview

Remove "NEW " prefixes from Getting Started template. (I think they are test cruft.)

> [!NOTE]
> If this text was actually a stakeholder request, please let me know because I should discuss the change in that realm first.

## Related

- fixes #281

## Changes

- **removed** bits of user-facing text

## Testing

1. Visit/Create a page with the "Getting Started" template.
2. Verify the text "NEW " is not present.

## UI

| current | goal | dev server |
| - | - | - |
| <img width="768" alt="current" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/e370f023-8683-49e6-b508-e4c0ad259987"> | <img width="768" alt="goal" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/0976c7ab-0f14-46dc-ab96-b8f05a554a89"> | <img width="768" alt="dev server" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/2e2f8e7f-e77b-40c0-8222-d970c7fb7392"> |
